apps/emqx/src/emqx_external_trace.erl

```diff
@@ -99,13 +96,6 @@ start_trace_send(Delivers, ChannelInfo) ->
 end_trace_send(Packets) ->
     ?with_provider(?FUNCTION_NAME(Packets), ok).
 
-event(Name) ->
```

Comment:
This was completely unused, so I see no reason in keeping this code. 
If we need add span events in future, it may anyway be implemented somehow differently.